websites/A/Animes House/dist/metadata.json

```diff
@@ -11,7 +11,7 @@
   },
   "url": "animeshouse.net",
   "version": "1.0.3",
```

Comment:
Increase the version

Reply:
corrected, I ended up forgetting this detail, sorry

Reply:
corrected, I ended up forgetting this detail, sorry